iec104.cpp

```diff
@@ -18,6 +18,20 @@
 #include <iostream>
 #include <utility>
 
+// Qualifier of command
+#define NoAddDefinition 0
```

Comment:
Why didn't you use static variables instead of #define?

Reply:
I agree, why not using simple class var ?